### PR TITLE
[FAU-423] Use `<TextControl>` for duration of studies

### PR DESCRIPTION
**Pull request by tyrann0us**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

**What is the current behavior?** (You can also link to an open issue here)
https://inpsyde.atlassian.net/browse/FAU-423
"Duration of studies in semester" field is a multi-line input (`<TextareaControlFormField>`).

**What is the new behavior (if this is a feature change)?**
"Duration of studies in semester" field is a single-line input (`<TextControl>`).

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No, existing content will be kept.

Before:
Bildschirmfoto 2024-05-03 um 11 46 20

After:
Bildschirmfoto 2024-05-03 um 11 48 00

**Other information**:
There's currently an unrelated error preventing any input ([source](https://inpsyde.slack.com/archives/C045KFJ9MB5/p1714728160353629)). However, this shouldn't affect this change (or vice versa).

### CODE REVIEW

File: resources/ts/components/DegreeProgramEditForm/General.tsx [hunk 1]
```diff
@@ -432,7 +432,7 @@ const General = () => {
 						fill="full"
```
Comment (amiut) [inline resources/ts/components/DegreeProgramEditForm/General.tsx:432]: Does it make sense to make the layout half/half with previous videos field?

**Review by amiut — APPROVED** [review]
Thanks for taking care of it, LGTM, left a small (unrelated) suggestion but nothing that prevents me from approving.

File: resources/ts/components/DegreeProgramEditForm/General.tsx (hunk 1 above)
Comment (tyrann0us) [reply resources/ts/components/DegreeProgramEditForm/General.tsx:432]: It does make sense, but it is out of scope. 😉

**Review by zhyian — APPROVED** [review]
Thanks!

**Review by shvlv — APPROVED** [review]

**Pull request merged**